pkgs/games/sopwith/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+, glib
+, SDL2
+, libGL
+, pkg-config
+, autoconf
+, automake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sopwith";
+  version = "2.3.0";
+
+  src = fetchurl {
+    url = "https://github.com/fragglet/sdl-sopwith/archive/refs/tags/sdl-sopwith-${version}.tar.gz";
+    hash = "sha256-3jIlB9AyWbevtYGEAIlNH6GIUKutEMRtRzay39ByMdg=";
+  };
```

```suggestion
  src = fetchFromGitHub {
    owner = "fragglet";
    repo = "sdl-sopwith";
    rev = "refs/tags/sdl-sopwith-${version}";
    hash = "sha256-C0OpFA3q3K5lgoVe0F03/zXlNba/zW7YEIH+2BV/nCI=";
  };
```
fetchurl is not stable fetching this url from github as the hash can change. use fetchFromGitHub instead.